src/services/repo/hooks.spec.tsx

```diff
@@ -266,5 +305,60 @@ describe('useRepoBackfilled', () => {
         )
       })
     })
+
+    describe('can throw errors', () => {
+      beforeAll(() => {
+        console.error = () => {}
+      })
+
+      afterAll(() => {
+        jest.resetAllMocks()
+      })
+      it('can return unsuccessful parse error', async () => {
+        setup({ isUnsuccessfulParseError: true })
+        const { result } = renderHook(() => useRepoBackfilled(), {
+          wrapper: wrapper(),
+        })
+
+        await waitFor(() => expect(result.current.isError).toBeTruthy())
+        await waitFor(() =>
+          expect(result.current.error).toEqual(
+            expect.objectContaining({
+              status: 404,
+            })
+          )
+        )
+      })
+      it('can return not found error', async () => {
+        setup({ isNotFoundError: true })
+        const { result } = renderHook(() => useRepoBackfilled(), {
+          wrapper: wrapper(),
+        })
+
+        await waitFor(() => expect(result.current.isError).toBeTruthy())
+        await waitFor(() =>
+          expect(result.current.error).toEqual(
+            expect.objectContaining({
+              status: 404,
+            })
+          )
+        )
+      })
+      it('can return owner not activated error', async () => {
```

Comment:
Just me being a little picky, moving forward if you're working on tests do you mind adding in an empty line in between these test blocks, I feel like it makes things a lot cleaning when you're working around the tests and "mini-mizing" them in your editor.

Reply:
For sure, I think stylistic things like this could be brought up in our weekly meeting as well if we want to make team norms around them!